packages/@aws-cdk/aws-codepipeline/lib/pipeline.ts

```diff
@@ -245,6 +246,37 @@ export class Pipeline extends cdk.Construct implements events.IEventRuleTarget {
     this.stages.splice(index, 0, stage);
   }
 
+  // ignore unused private method (it's actually used in Stage)
+  // @ts-ignore
+  private _generateOutputArtifactName(stage: actions.IStage, action: actions.Action): string {
+    // for now, just return a generic output artifact,
+    // ignoring the names of both the Stage, and the Action
+    return 'Artifact_' + (++this.artifactsCounter);
+  }
+
+  // ignore unused private method (it's actually used in Stage)
+  // @ts-ignore
+  private _findInputArtifactFor(stage: actions.IStage, action: actions.Action): actions.Artifact {
```

Comment:
add doc that describes what this method is doing

Reply:
Done.